digits/dataset/images/classification/forms.py

```diff
@@ -38,6 +38,158 @@ def validate_backend(form, field):
             tooltip='Compressing the dataset may significantly decrease the size of your database files, but it may increase read and write times.',
             )
 
+    ### Data augmentation - rotation 
+    has_augmentation_rotation = utils.forms.BooleanField('Perform rotations on images',
+            default = False,
+            tooltip = "You can augment your dataset by performing rotation on images.",
+            validators=[ 
+                ]
+            )
+
+    augmentation_rotation_angle_min = utils.forms.IntegerField(u'angle min',
+            default=-90,
+            validators=[ 
+                validators.NumberRange(min=-360, max=360)
+                ],
+            tooltip = "The minimum rotation angle that can be performed during augmentation."
+            )
+
+    augmentation_rotation_angle_max = utils.forms.IntegerField(u'angle max',
+            default=90,
+            validators=[ 
+                validators.NumberRange(min=-360, max=360)
+                ],
+            tooltip = "The maximum rotation angle that can be performed during augmentation."
+            )
+
+    augmentation_rotation_probability = utils.forms.FloatField(u'probability',
+            default=0.75,
+            validators=[ 
+                validators.NumberRange(min=0.0, max=1.0)
+                ],
+            tooltip = "The probability for an image to be rotated during augmentation."
+            )
+
+    ### Data augmentation - hue 
+    has_augmentation_hue = utils.forms.BooleanField('Perform hue modulation on images',
+            tooltip="You can augment your dataset by performing hue modulation on images.",
+            default=False,
+            validators=[ 
+                ]
+            )
+ 
+    augmentation_hue_angle = utils.forms.FloatField(u'angle',
```

Comment:
I think he means the `augmentation_hue_angle` field. Why does a hue have an angle?